test-suite/src/test/scala/org/scalajs/testsuite/jsinterop/TupleTest.scala

```diff
@@ -0,0 +1,42 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js Test Suite        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013, LAMP/EPFL        **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+package org.scalajs.testsuite.jsinterop
+
+import org.scalajs.jasminetest.JasmineTest
+
+import scala.scalajs.js
+
+object TupleTest extends JasmineTest {
+
+  describe("scala.scalajs.js.Tuple") {
+
+    it("should provide an equivalent of scala tuple") {
+      val obj: js.Tuple2[Int, String] = (42, "foobar")
```

Comment:
Oh, and also one test asserting that a newly created `js.Tuple2(42, "foobar")` is actually a JS array, by testing `(obj: Any).isInstanceOf[js.Array[_]]`.